src/data/faq.json

```diff
@@ -2171,66 +2171,304 @@
+                },
+                {
+                    "term": "Short Lived Random Bluetooth ID",
+                    "anchor": "short_lived_random_bluetooth_id",
+                    "description": "Unique key generated several times an hour, derived from the random device key and exchanged between neighboring smartphones via Bluetooth."
+                },
                 {
                     "term": "Signature",
                     "anchor": "signature",
-                    "description": "A digital signature is a signature that is created by a cryptographic function from a digital document. Digital signatures provide the following properties: <ul><li>Non-repudiation: The issuer cannot retrospectively deny signing the document.</li><li>Integrity: Altering the document invalidates the signature.</li><li>Authenticity: It is clear who signed the document.</li></ul> <br/><br/> As a consequence, the signature must fulfill the following requirements: <br/><ul><li>The signature must be computationally verifiable </li><li>Verifying the signature must be easy - forging a valid signature must be hard</li><li>The signature is only valid for the signed document and no other documents </li><li>The issuer must not be able to deny signing the document.</li></ul>"
+                    "description": "A digital signature is a signature that is created by a crytographic function from a digital document. Digital signatures provide the following properties: <ul><li>Non-repudiation: The issuer cannot retrospectively deny signing the document.</li><li>Integrity: Altering the document invalidates the signature.</li><li>Authenticity: It is clear who signed the document.</li></ul> As a consequence, the signature must fulfill the following requirements: <ul><li>The signature must be computationally verifiable </li><li>Verifying the signature must be easy - forging a valid signature must be hard</li><li>The signature is only valid for the signed document and no other documents </li><li>The issuer must not be able to deny signing the document.</li></ul>"
+                },
+                {
+                    "term": "Standardized Name",
+                    "anchor": "standardized_name",
+                    "description": "Standardized names refer to names that are additionally presented with capital letters in standardized Latin script due to their country-specific spelling. This can occur, for example, with Cyrillic, Arabic, but also German script. For example, a \"Ö\" becomes a \"OE\"."
                 }
             ],
             "T": [
+                {
+                    "term": "TAN",
+                    "anchor": "tan",
+                    "description": "10-digit belly letter number combination for reporting positive test results. To request this via the TAN hotline. <b>National:</b> 0800 7540002 and <b>international:</b> +49 30 498 75402"

```

Comment:
remove "belly"! It is a translation of a German typo (Bauchstaben instead of Buchstaben)!

Reply:
Will be changed in the next commit, thanks!